lua/octo/commands.lua

```diff
@@ -0,0 +1,277 @@
+end
+
+
+function M.new_comment()
+	local bufnr = api.nvim_get_current_buf()
+	local iid = api.nvim_buf_get_var(bufnr, 'iid')
+	local number = api.nvim_buf_get_var(bufnr, 'number')
+	local repo = api.nvim_buf_get_var(bufnr, 'repo')
+
+	if not iid or not number or not repo then
+		api.nvim_err_writeln('Buffer is not linked to a GitHub issue')
+		return
+	end
+
+  gh.run({
+    args = {
+      'api', '-X', 'POST',
+      '-f', format('body=%s', NO_BODY_MSG),
+      format('repos/%s/issues/%s/comments', repo, number)
+    };
+    cb = function(output)
+      local comment = json.parse(output)
+      if nil ~= comment['issue_url'] then
+        octo.write_comment(bufnr, comment)
+        vim.fn.execute('normal! Gkkk')
+      end
+    end
+  })
+end
+
+function M.new_issue(repo)
+  if not repo then repo = util.get_remote_name() end
+  gh.run({
+    args = {
+      'api', '-X', 'POST',
+      '-f', format('title=%s', 'title'),
+      '-f', format('body=%s', NO_BODY_MSG),
+      format('repos/%s/issues', repo)
+    };
+    cb = function(output)
+      octo.create_issue_buffer(json.parse(output), repo)
+    end
+  })
+end
+
+function M.change_issue_state(state)
+	local bufnr = api.nvim_get_current_buf()
+	local number = api.nvim_buf_get_var(bufnr, 'number')
+	local repo = api.nvim_buf_get_var(bufnr, 'repo')
+
+	if not state then
+		api.nvim_err_writeln('Missing argument: state')
+		return
+	end
+
+	if not number or not repo then
+		api.nvim_err_writeln('Buffer is not linked to a GitHub issues')
+		return
+	end
+
+  gh.run({
+    args = {
+      'api', '-X', 'PATCH',
+      '-f', format('state=%s', state),
+      format('repos/%s/issues/%s', repo, number)
+    };
+    cb = function(output)
+      local resp = json.parse(output)
+      if state == resp['state'] then
+        api.nvim_buf_set_var(bufnr, 'state', resp['state'])
+        -- TODO: do not reload issue, just header
+        M.get_issue(repo, resp['number'])
+        print('Issue state changed to: '..resp['state'])
+      end
+    end
+  })
+end
+
+function M.issue_action(action, kind, value)
+  if vim.bo.ft ~= 'octo_issue' then api.nvim_err_writeln('Not in octo buffer') return end
+
+  local number_ok, number = pcall(api.nvim_buf_get_var, 0, 'number')
+  if not number_ok then api.nvim_err_writeln('Missing octo metadata') return end
+  local repo_ok, repo = pcall(api.nvim_buf_get_var, 0, 'repo')
+  if not repo_ok then api.nvim_err_writeln('Missing octo metadata') return end
+
+  vim.validate{
+    action = {action,
+      function(a)
+        return vim.tbl_contains({'add', 'remove'}, a)
+      end,
+      'add or remove'
+    },
+    kind = {kind,
+      function(a)
+        return vim.tbl_contains({'assignees', 'labels', 'requested_reviewers'}, a)
+      end,
+      'assignees, labels or requested_reviewers'
+    },
+  }
+
+  local endpoint
+  if kind == 'requested_reviewers' then
+    endpoint = 'pulls'
+  else
+    endpoint = 'issues'
+  end
```

Comment:
```suggestion
endpoint = kind == 'requested_reviewers' and 'pulls' or 'issues'
```
This is an extra comment with this here suggestion